lib/config.js

```diff
@@ -4,6 +4,37 @@ var parseDatabaseUrl = require('parse-database-url');
 var dbmUtil = require('db-migrate-shared').util;
 var log = require('db-migrate-shared').log;
 
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
```

Comment:
As this patch only seems to relate to AWS we may shouldn't place it in the config.js file. It is used all around the project though.